modules/ROOT/pages/ldap/ldap-connector.adoc

```diff
@@ -291,7 +295,7 @@ output application/java
 ----
 +
 . Drag Add entry operation of the LDAP connector next to the DataWeave component to add the LDAP Entry.
-. Configure the LDAP connector by adding a new LDAP Global Element. Click the plus sign next to the Extension configuration field.
+. Configure the LDAP connector by adding a new LDAP Global Element. Click the plus sign next to the Connector configuration field.
```

Comment:
We should be saying "LDAP Connector", but you might not have the bandwidth to make this change throughout the doc. Also, "Global Element" should be lowercase.

*Connector configuration* field